frontend/stubs/stubs_handler.py

```diff
@@ -30,8 +29,9 @@ def infer_file(tree, context, solver, used_names):
                                node.name in used_names)]
 
         for stmt in relevant_nodes:
-            infer(stmt, context, solver)
+            infer_func(stmt, context, solver)
 
-    def infer_all_files(self, context, solver, used_names):
-        for tree in self.asts:
-            self.infer_file(tree, context, solver, used_names)
+    @classmethod
```

Comment:
Hm. I think I would have made this a normal method and made asts a field of each object. Is there a reason for not doing that? For example, if we do inference for several programs in a row (e.g. in a test suite or from an IDE), do we want to do this every time or only once? If we might want to do it every time, asts should be an object attribute IMO.